Makefile.am

```diff
@@ -79,6 +81,11 @@ dist_polkitrules_DATA = \
 	10-allow-mounting-endless-image-device.rules \
 	$(NULL)
 
+tmpfilesddir = $(libdir)/tmpfiles.d
```

Comment:
We can't use `$(libdir)` here unfortunately. See e.g. this commit: https://github.com/endlessm/eos-metrics-instrumentation/commit/d4953db3bab8ffa55694849acf884c704ae144bb#diff-8d777f385d3dfec8815d20f7496026dc

Reply:
Ah, frustrating. Thanks for catching!